### DISCUSSION TITLE
Add basic-authentication support for /admin/clients [POST, PUT, PATCH, DELETE] methods

### DISCUSSION THREAD

**Discussion by rubanraj-r** [ory/hydra#3899] [Ideas]
Add basic-authentication support for /admin/clients [POST, PUT, PATCH, DELETE] methods to secure the system from unnecessary  or unauthentic creation/updation/deletion of the clients.

**Comment by vinckr** [comment]
Hello @rubanraj-r 

Securing the admin APIs is out of scope for the Ory Hydra project itself. 

Here is an example how you can secure the admin API: https://www.ory.sh/docs/hydra/self-hosted/go#with-authentication
